examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -20,34 +21,48 @@ if (location.hash.length === 0) {
 const containerId = location.hash.substring(1);
 document.title = containerId;
 
-const dataObjectId = "dice";
+// Define the configuration of our Container.
+// This includes the DataObjects we support and any initial DataObjects we want created
+// when the container is first created.
+export const containerConfig = {
+    name: "dice-roller-container",
+    initialObjects: {
+        /* [id]: DataObject */
+        kvp: KeyValueDataObject,
```

Comment:
These represent individual objects rather than types, right?  As in you could have multiple properties in `initialObjects` e.g. with type `KeyValueDataObject`, or is it more like how registry entries work?

Reply:
correct. `initialObjects` are individual objects that will be available to the developer when the container is first loaded. You can have multiple objects of the same type.